packages/cli/lib/LocalDevManager.js

```diff
@@ -298,16 +298,20 @@ class LocalDevManager {
 
   async checkPublicAppInstallation() {
     const {
-      isInstalledWithScopeGroups: isInstalled,
+      isInstalledWithScopeGroups,
+      previouslyAuthorizedScopeGroups,
     } = await this.getActiveAppInstallationData();
 
-    if (!isInstalled) {
+    const isReinstall = previouslyAuthorizedScopeGroups.length > 0;
```

Comment:
Can you explain what the `previouslyAuthorizedScopeGroups` and `isInstalledWithScopeGroups` contain? I just want to make sure I'm understanding how this works now.

Reply:
@brandenrodgers `previouslyAuthorizedScopeGroups` is the list of scopes you authorized the last time you installed the app. The field having a length of more than zero means that at some point you installed the app in the past. `isInstalledWithScopeGroups` is true when you've authorized the _current_ set of required scopes for the app. This can be false if the required scopes have changes since you last installed

Reply:
Thanks, the logic here looks perfect then 👌